### ISSUE TITLE
Release TRIAL?

### ISSUE THREAD

**Issue by eserte** [barefootcoder/dist-zilla-pluginbundle-barefoot#14]
http://matrix.cpantesters.org/?dist=Dist-Zilla-PluginBundle-BAREFOOT%200.06_01-TRIAL looks better than http://matrix.cpantesters.org/?dist=Dist-Zilla-PluginBundle-BAREFOOT%200.06 (and as a plus, your distribution won't show up on the beforemaintrelease page as a possible perl regression victim)

**Comment by barefootcoder** [comment]
Yeah, it's definitely better, but there was still a problem, and I was hoping to have time to fix that before upgrading to full release.  But I'm not sure I will now.  Let me take a closer look this weekend and see if it's better to just kick it loose as is.

Thanx for the reminder.

**Comment by eserte** [comment]
Well, 0.06_01-TRIAL does not look that good anymore:
```
t/require.t .. ok
[@BAREFOOT/NextRelease] failed to find Changes in the distribution at /usr/perl5.22.2t/lib/site_perl/5.22.2/amd64-freebsd-thread-multi/Moose/Meta/Method/Delegation.pm line 110.
t/tdist.t .... 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

**Comment by barefootcoder** [comment]
Finally getting back around to this one as well.  It's so fragile, until I can resolve the class/package thing, that I got scared to fuck with it for a while there. :-)

Anyhow, I can't see why some smokers pass and some fail, unless it has to do with the CPAN Reporter version (that was the only thing I noticed, but it's a tenuous pattern at best).  So I've added some `diag` debugging and released a new 0.06_02.  Let's see how well _that_ goes ...

**Comment by eserte** [comment]
http://analysis.cpantesters.org/solved?distv=Dist-Zilla-PluginBundle-BAREFOOT-0.06_01-TRIAL suggests that it fails with Dist::Zilla >= 6.
